persist/sqlite/hosts_test.go

```diff
@@ -423,8 +423,11 @@ func TestQueryHosts(t *testing.T) {
 		{
 			name: "v1 min duration 1000",
 			query: explorer.HostQuery{
-				V2:          &falseBool,
-				MinDuration: 1000,
+				V2: &falseBool,
+				MinDuration: func() *uint64 {
```

Comment:
I'd inline a helper function once and call that instead of inlining this 4 times.